libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -931,11 +931,19 @@ export namespace EmbeddedKpiDashboard {
         FilterContextChangedBody
     >;
 
+    /**
+     * @public
+     */
+    export type DrillToUrlFilters = Array<
+        EmbeddedGdc.DashboardDateFilter | EmbeddedGdc.IDashboardAttributeFilter
+    >;
+
     /**
      * @public
      */
     export interface IDrillToUrlStartedDataBody {
         id: string;
+        filters: DrillToUrlFilters;
```

Comment:
please add few lines of docs for this new prop. write what are the values of these filters (e.g. values of dashboard filters set when user triggered the drill).

also give hint on the existence of the type-guards. "You can use the typeguards defined in EmbeddedGdc namespace to test the type of the filter. "